src/node_os.cc

```diff
@@ -271,6 +271,29 @@ static void GetInterfaceAddresses(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+static void GetHomeDirectory(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+#ifdef _WIN32
+  char buf[MAX_PATH];
+#else
+  char buf[PATH_MAX];
+#endif
```

Comment:
node.h defines PATH_MAX on Windows so the #ifdef is probably not necessary here.